StRoot/StBFChain/StBFChain.cxx

```diff
@@ -638,11 +638,14 @@ Int_t StBFChain::Instantiate()
     }
 
     if (maker == "StStrangeMuDstMaker" && GetOption("CMuDST")&& GetOption("StrngMuDST") ) {
-      TString cmd(Form("StStrangeMuDstMaker *pSMMk = (StStrangeMuDstMaker*) %p;",mk));
-      cmd += "pSMMk->DoV0();";                                 // Set StrangeMuDstMaker parameters
-      cmd += "pSMMk->DoXi();";
-      cmd += "pSMMk->DoKink();";
-      cmd += "pSMMk->SetNoKeep();";                            // Set flag for output OFF
+printf ("UUU Mk = %p\n",mk);
```

Comment:
This introduces a change to the StStrangeMuDstMaker configuration... and also creates meaningless output.

The old version needs to be restored.